index_V1

```diff
@@ -85,25 +217,62 @@
 				
 				<div class="table_2" style="margin: 2vw 15vw;">
 					
-					<div class="text" style="grid-column: 1; background-color: grey; margin: 0vw 1vw;"></div>
+					<div class="text" style="grid-column: 1; background-color: white; margin: 0vw 1vw;">
+					
+						<h3 style="font-size: 2.2vw; font-family: 'Noto Sans JP', sans-serif;">FOR YOU</h3>
+						
+						<p>Suggested Playlists <br>we recommend you <br>to check out!</p>
+					
+					</div>
+					
+					<div onclick="location.href='query_greatest.php';" class="circle_1" style="cursor: pointer; grid-column: 2; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s; overflow: hidden;">
+						
+						<img src="images/img_10.jpg" img draggable="false" alt="Blue" style="width: 12vw; height: 12vw;">
+					
+					</div>
 					
-					<div onclick="location.href='query_1.php';" class="circle_1" style="cursor: pointer; grid-column: 2; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s;"></div>
+					<div onclick="location.href='query_gym.php';" class="circle_2" style="cursor: pointer; grid-column: 3; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s; overflow: hidden;">
+						
+						<img src="images/img_13.jpg" img draggable="false" alt="Blue" style="width: 12vw; height: 12vw;">
```

Comment:
The Gym Music Query has now been updated to contain a custom gym image for the Album Cover